src/bundle/Resources/public/scss/_tag.scss

```diff
@@ -24,6 +24,7 @@
     }
 
     &__remove-btn {
+        display: none;
```

Comment:
Wouldn't such a change make this an `ibexa-badge`? 🤔

Reply:
Well, yes, but actually no.

It looks the same way as badge when `is_deletable` set to false, but when it can be deleted it's not. And I don't think it would be wise to change whole rendered/used component based on only `is_deletable` param.